itchiodl/utils.py

```diff
@@ -48,10 +48,10 @@ def clean_path(path):
     return path
 
 
-def md5sum(path):
+def md5sum(pathname):
```

Comment:
This code was deliberately changed to use pathlib to avoid the string cast

Reply:
Reverted this change